src/Microsoft.AspNetCore.SystemWebAdapters/Adapters/IHttpRequestAdapterFeature.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#if NET6_0_OR_GREATER
```

Comment:
Can we just use netcoreapp here? Do we ever build for NET5_0 or lower?

Reply:
Sure - didn't realize NETCOREAPP would cover .NET 6+ :)